README.rst

```diff
@@ -11,33 +13,51 @@ each using a different `client library`_ to communicate with CrateDB_.
 Prerequisites
 =============
 
-You need install and run CrateDB 0.54.1 or higher before you set things up.
+- You need install and run CrateDB 0.54.1 or higher before you set things up.
+- If you're using the Java backend, you will need 0.57.0 or higher.
+- You will also need to install crash_, the CrateDB CLI tool, to work with CrateDB
+  on the command line.
+- See the CrateDB `Getting Started`_ guide for help.
 
-If you're using the Java backend, you will need 0.57.0 or higher.
+Setup
+=====
 
-You will also need to install crash_ (the CrateDB CLI tool) to work with CrateDB
-on the command line.
+Once a CrateDB instance is running, create the required schema and import the
+country data::
 
-See the CrateDB `Getting Started`_ guide for help.
+    crash < sql/schemas.sql
+    crash -c "COPY guestbook.countries FROM '$(pwd)/sql/countries.json' RETURN SUMMARY;"
 
-Setup
-=====
+If you choose to use Docker for running CrateDB on your workstation, those
+commands outline the process::
 
-Once CrateDB instance is running, create the required schema and import the
-country data:
+    # Define CrateDB version.
+    export CRATEDB_VERSION=5.1.2
+    export CRATEDB_IMAGE=crate:${CRATEDB_VERSION}
```

Comment:
Is there a good reason not to use `latest` ❓

Reply:
No. Also fixed with 79278325.